src/server/controllers/usersController.js

```diff
@@ -1,12 +1,33 @@
 
 const User = require('../models/user');
+var sesh;
```

Comment:
I'm not sure what's the purpose of the `sesh` variable which isn't being used.

Reply:
This is for the session management after logging in, but it is not implemented yet